task_test.go

```diff
@@ -142,6 +144,14 @@ func (s *TaskSuite) TestBuildArgsStaticAndFile() {
 	s.NoError(err)
 }
 
+func (s *TaskSuite) TestBuildkitSecrets() {
+	s.req.Config.ContextDir = "testdata/buildkit-secret"
+	s.req.Config.BuildkitSecrets["secret"] = "testdata/buildkit-secret/secret"
```

Comment:
(2/2) - so can this just set the whole value instead?

```suggestion
	s.req.Config.BuildkitSecrets = map[string]string{"secret": "testdata/buildkit-secret/secret"}
```

Reply:
Ahh, yep, that makes sense. Still getting my head around Go. I'll make this change now and test it in our environment.